### PR TITLE
kotlin cleanups: nullability

### PR DESCRIPTION
**Pull request by david-allison**
Nad a NRE in test code, and these calls looked dodgy.

They weren't the issue, but let's fix them to make it easier for the next dev.

Used conventions from the surrounding code when handing `context == null`

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/pages/PageChromeClient.kt [hunk 1]
```diff
@@ -24,22 +24,22 @@ import com.ichi2.anki.UIUtils
...
 open class PageChromeClient : WebChromeClient() {
     override fun onJsAlert(
-        view: WebView?,
+        view: WebView,
         url: String?,
         message: String?,
         result: JsResult?
     ): Boolean {
-        UIUtils.showThemedToast(view?.context, message, shortLength = true)
+        UIUtils.showThemedToast(view.context, message ?: "", shortLength = true)
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/pages/PageChromeClient.kt:32]: About the nullability change on `message`, I think that is better better to not show a toast than showing an empty one

**Review by BrayanDSO — CHANGES_REQUESTED** [review]
Almost looks good to me

File: AnkiDroid/src/main/java/com/ichi2/anki/pages/PageChromeClient.kt (hunk 1 above)
Comment (david-allison) [reply AnkiDroid/src/main/java/com/ichi2/anki/pages/PageChromeClient.kt:32]: Unsure on this one. If someone performed `alert()` in JS (presumably similar), we would want an alert to be shown

**Review by BrayanDSO — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
both commits LGTM

**Pull request merged**